it.js

```diff
@@ -62,6 +62,7 @@ export const cuss = {
   'bel bocconcino': 1,
   belini: 2,
   belino: 2,
+  dio: 2,
```

Comment:
I don’t think `god` is always a slur. https://github.com/words/cuss/blob/e09ba41d2b2398057a5c5cbaaaf235ae51ba468d/index.js#LL674C3-L674C6

Reply:
I edit the fork, i added new words and I reduce "dio" word to 1.